lib/graphql/authorization.rb

```diff
@@ -71,7 +71,13 @@ def final_value(memo)
           end
           context = memo[:context]
           err = InaccessibleFieldsError.new(fields: fields, irep_nodes: nodes, context: context)
-          context.schema.inaccessible_fields(err)
+
+          # TODO: fix
+          if context.schema.respond_to?(:inaccessible_field)
+            context[:inaccessible_nodes] = memo[:inaccessible_nodes]
```

Comment:
This can be added to the context regardless. The only conditional thing should be skipping `inaccessible_fields`.